Source/carthage/Checkout.swift

```diff
@@ -52,7 +52,7 @@ public struct CheckoutOptions: OptionsType {
 		return create
 			<*> m <| Option(key: "use-ssh", defaultValue: false, usage: "use SSH for downloading GitHub repositories")
 			<*> m <| Option(key: "use-submodules", defaultValue: false, usage: "add dependencies as Git submodules")
-			<*> m <| Option(key: "use-binaries", defaultValue: true, usage: "check out dependency repositories even when prebuilt frameworks exist, disabled if --use-submodules option is present" + useBinariesAddendum)
+			<*> m <| Option(key: "use-binaries", defaultValue: false, usage: "check out dependency repositories even when prebuilt frameworks exist, disabled if --use-submodules option is present" + useBinariesAddendum)
```

Comment:
I'm not familiar with this aspect of `Commandant`: will `no-use-binaries` be a valid option (though redundant) after this change?